### PR TITLE
Fund GHO Liquidity Committee

### PR DESCRIPTION
**Pull request by defijesus**

### CODE REVIEW

File: src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/AaveV3_Ethereum_FundGHOLiquidityCommittee_20230926.sol [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ * @title Fund GHO Liquidity Committee
+ * @author TokenLogic
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x219cf8fbfa806b396728b7bf11e635ac4833ce92c9ea8e34f24a686e3cf0d132
+ * - Discussion: https://governance.aave.com/t/arfc-treasury-manage-gho-liquidity-committee/14914
+ */
+contract AaveV3_Ethereum_FundGHOLiquidityCommittee_20230926 is IProposalGenericExecutor {
+  function execute() external {
+    uint256 daiAmount = 406_000e18;
+    uint256 awethAmount = 5 ether;
+    address daiOracle = 0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9;
```
Comment (sakulstra) [inline src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/AaveV3_Ethereum_FundGHOLiquidityCommittee_20230926.sol:41]: fine, why not just use the oracle from `AaveV3EthereumAssets.DAI_ORACLE`

File: src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/AaveV3_Ethereum_FundGHOLiquidityCommittee_20230926.sol [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+/**
+ * @title Fund GHO Liquidity Committee
+ * @author TokenLogic
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x219cf8fbfa806b396728b7bf11e635ac4833ce92c9ea8e34f24a686e3cf0d132
+ * - Discussion: https://governance.aave.com/t/arfc-treasury-manage-gho-liquidity-committee/14914
+ */
+contract AaveV3_Ethereum_FundGHOLiquidityCommittee_20230926 is IProposalGenericExecutor {
+  function execute() external {
+    uint256 daiAmount = 406_000e18;
```
Comment (sakulstra) [inline src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/AaveV3_Ethereum_FundGHOLiquidityCommittee_20230926.sol:39]: all of these vars could be `public constant` so it is easier to review onchain.

File: src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+The GLC's responsibilities include distributing rewards across liquidity pools and strategies that support GHO. The GLC is initially established for a 3-month period.
+
+## Specification
+
+This proposal encompasses the following actions:
+
+* Swap 438,000 aEthDAI to GHO via the Aave Swap Contract
+* Transfer 5 ETH and 438,000 units of GHO to the GLC SAFE address
+
+GLC SAFE Address: `0x205e795336610f5131Be52F09218AF19f0f3eC60`
```
Comment (sakulstra) [inline src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md:27]: better always put etherscan links for addresses
```suggestion
GLC SAFE Address: [0x205e795336610f5131Be52F09218AF19f0f3eC60](https://etherscan.io/address/0x205e795336610f5131Be52F09218AF19f0f3eC60#code)
```

File: src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+* Growing GHO liquidity
+
+The GLC's responsibilities include distributing rewards across liquidity pools and strategies that support GHO. The GLC is initially established for a 3-month period.
+
+## Specification
+
+This proposal encompasses the following actions:
+
+* Swap 438,000 aEthDAI to GHO via the Aave Swap Contract
+* Transfer 5 ETH and 438,000 units of GHO to the GLC SAFE address
```
Comment (sakulstra) [inline src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md:25]:
```suggestion
* Transfer 5 ETH 
* Approve 438,000 units of GHO to the GLC SAFE address
```

File: src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* Improving the GHO peg
+* Growing GHO liquidity
+
+The GLC's responsibilities include distributing rewards across liquidity pools and strategies that support GHO. The GLC is initially established for a 3-month period.
+
+## Specification
+
+This proposal encompasses the following actions:
+
+* Swap 438,000 aEthDAI to GHO via the Aave Swap Contract
```
Comment (sakulstra) [inline src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md:24]:
```suggestion
* Initiate the Swap 438,000 aEthDAI to GHO via the Aave Swap Contract
```

File: src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+discussions: "https://governance.aave.com/t/arfc-treasury-manage-gho-liquidity-committee/14914"
+---
+
+## Simple Summary
+
+This publication proposes the creation and funding of the GHO Liquidity Committee (GLC) for an initial 3-month period.
+
+## Motivation
+
+GHO is currently trading below its peg, with liquidity spread across various pools. To address this, the GLC will receive a budget of 5 ETH and 438,000 units of GHO with the primary objectives of:
```
Comment (sakulstra) [inline src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md:13]: where does this number come from? proposal swaps 406k DAI to whatever currently is fair price - 50bps slippage, no?

File: src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md (hunk 5 above)
Comment (sakulstra) [reply src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md:24]: also the aip sais 438k - but forum and impelemntation say 406k? and the conditions of the swap could be more elaborate here.

**Review by sakulstra — DISMISSED** [review]

**Review by rustboyar — APPROVED** [review]

**Review by sakulstra — APPROVED** [review]

**Review by rustboyar — APPROVED** [review]

**Pull request merged**